src/library.js

```diff
@@ -398,11 +398,7 @@ addToLibrary({
   // ==========================================================================
 
 #if !STANDALONE_WASM
-  // TODO: There are currently two abort() functions that get imported to asm
-  // module scope: the built-in runtime function abort(), and this library
-  // function _abort(). Remove one of these, importing two functions for the
-  // same purpose is wasteful.
-  abort: () => {
```

Comment:
I think the TODO is still relevant, in that in theory we could imagine not going through this "trampoline" here. But closure will inline it anyhow so I'm not sure if it matters.

Reply:
My current thinking is that trampiline is useful since it injects the "native code aborted" message before calling the JS abort function.     So I don't think should be trying to remove it anymore.